app/src/components/ResourceInfoCard.tsx

```diff
@@ -22,50 +36,105 @@ const useStyles = createStyles(theme => ({
 
 export default function ResourceInfoCard({ resourceInfo, authoring }: ResourceInfoCardProps) {
   const { classes } = useStyles();
+  const ctx = trpc.useContext();
+  const [isConfirmationModalOpen, setIsConfirmationModalOpen] = useState(false);
+
+  const deleteMutation = trpc.draft.deleteDraft.useMutation({
+    onSuccess: () => {
+      notifications.show({
+        title: `Draft ${resourceInfo.resourceType} Deleted!`,
+        message: `Draft ${resourceInfo.resourceType}/${resourceInfo.id} successfully deleted`,
+        icon: <CircleCheck />,
+        color: 'green'
+      });
+      ctx.draft.getDraftCounts.invalidate();
+      ctx.draft.getDrafts.invalidate();
+    },
+    onError: e => {
+      notifications.show({
+        title: `Draft ${resourceInfo.resourceType} Deletion Failed!`,
+        message: `Attempt to delete draft of ${resourceInfo.resourceType}/${resourceInfo.id} failed with message: ${e.message}`,
+        icon: <AlertCircle />,
+        color: 'red'
+      });
+    }
+  });
+
   return (
-    <Paper className={classes.card} shadow="sm" p="md">
-      <Grid align="center">
-        <Grid.Col span={10}>
-          <div>
-            <Text size="lg" fw={700}>
-              {resourceInfo.name ? resourceInfo.name : `${resourceInfo.resourceType}/${resourceInfo.id}`}
-            </Text>
-          </div>
-          {resourceInfo.version ? (
-            <Text size="sm">
-              {`v${resourceInfo.version}`} {resourceInfo.status && <b> ({resourceInfo.status})</b>}
-            </Text>
-          ) : (
-            <Text size="sm">{resourceInfo.status && <b>Status: {resourceInfo.status}</b>}</Text>
-          )}
-          {resourceInfo.url && (
-            <Text size="sm">
-              {resourceInfo.url}
-              {resourceInfo.version && `|${resourceInfo.version}`}
-            </Text>
-          )}
-          {resourceInfo.identifier && (
-            <Text size="sm">
-              <i>Identifier: </i>
-              {resourceInfo.identifier}
-            </Text>
-          )}
-        </Grid.Col>
-        <Link
-          href={
-            authoring
-              ? `/authoring/${resourceInfo.resourceType}/${resourceInfo.id}`
-              : `/${resourceInfo.resourceType}/${resourceInfo.id}`
-          }
-          key={resourceInfo.id}
-        >
-          <Tooltip label={authoring ? 'Edit Draft Resource' : 'View Resource Contents'} openDelay={1000}>
-            <ActionIcon radius="md" size="md" variant="subtle" color="gray">
-              {authoring ? <Edit size="24" /> : <SquareArrowRight size="24" />}
-            </ActionIcon>
-          </Tooltip>
-        </Link>
-      </Grid>
-    </Paper>
+    <>
+      <ConfirmationModal
+        open={isConfirmationModalOpen}
+        onClose={() => setIsConfirmationModalOpen(false)}
+        title={'Delete Draft?'}
```

Comment:
This is totally subjective but I'm not a huge fan of the title in the delete confirmation modal. I think the modal text suffices for letting the user know what to do. I edited it and included the screenshot to show what it would look like without the title included! 

![Screenshot 2023-06-27 at 1 34 48 PM](https://github.com/projecttacoma/measure-repository/assets/98363677/cbe47e7c-a7ba-4292-b970-33503d520b08)

Reply:
I worked with this a bit and I think the modal looks best without the title since this the action is pretty simple here (deleting a resource) - I removed the title for now